libraries/tuttle/CMakeLists.txt

```diff
@@ -198,12 +196,13 @@ if (TuttleBoost_FOUND)
         DESTINATION lib/python${PYTHONLIBS_VERSION_STRING}/site-packages/pySequenceParser)
     install(FILES ${CMAKE_BINARY_DIR}/libraries/tuttle/sequenceParser.py 
         DESTINATION lib/python${PYTHONLIBS_VERSION_STRING}/site-packages/pySequenceParser)
-    file(WRITE "__init__.py" "")
-    install(FILES "__init__.py" 
-        DESTINATION lib/python${PYTHONLIBS_VERSION_STRING}/site-packages/pySequenceParser)
+    install(CODE "file(WRITE ${CMAKE_INSTALL_PREFIX}/lib/python${PYTHONLIBS_VERSION_STRING}/site-packages/pySequenceParser/__init__.py)")
```

Comment:
To avoid to create a __init__.py file next to libraries/tuttle/CMakeLists.txt when build TuttleOFX.
Now the __init__.py is only created in installation directory.